ocrd_keraslm/wrapper/ocrd-tool.json

```diff
@@ -0,0 +1,24 @@
+{
+  "git_url": "https://github.com/OCR-D/ocrd_keraslm",
+  "version": "0.0.1",
+  "tools": {
+    "ocrd-keraslm-rate": {
+      "executable": "ocrd-keraslm-rate",
+      "categories": [
+        "Text recognition and optimization"
+      ],
+      "steps": [
+        "recognition/text-recognition"
+      ],
+      "description": "Rate text with keras",
+      "parameters": {
+        "model": {
+          "type": "string"
```

Comment:
`"required": true` or provide a default (which doesn't make sense in this case, unless we bundle a sample model with the tool).